src/status/condition/mod.rs

```diff
@@ -108,6 +108,98 @@ pub struct ClusterCondition {
     pub type_: ClusterConditionType,
 }
 
+impl std::fmt::Display for ClusterCondition {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let out = match self.type_ {
+            ClusterConditionType::Available => match self.status {
+                ClusterConditionStatus::True => "Available",
+                ClusterConditionStatus::False => "Unavailable",
+                ClusterConditionStatus::Unknown => "Availability unknown",
+            },
+            ClusterConditionType::Degraded => match self.status {
+                ClusterConditionStatus::True => "Degraded",
+                ClusterConditionStatus::False => "Not degraded",
+                ClusterConditionStatus::Unknown => "Degradation unknown",
+            },
+            ClusterConditionType::Progressing => match self.status {
+                ClusterConditionStatus::True => "Progressing",
+                ClusterConditionStatus::False => "Not progressing",
+                ClusterConditionStatus::Unknown => "Progression unknown",
+            },
+            ClusterConditionType::ReconciliationPaused => match self.status {
+                ClusterConditionStatus::True => "Reconciling",
+                ClusterConditionStatus::False => "Not reconciling",
+                ClusterConditionStatus::Unknown => "Reconciliation unknown",
+            },
+            ClusterConditionType::Stopped => match self.status {
+                ClusterConditionStatus::True => "Stopped",
+                ClusterConditionStatus::False => "Running",
+                ClusterConditionStatus::Unknown => "Stopped status unknown",
+            },
+        };
+
+        out.fmt(f)
+    }
+}
+
+impl ClusterCondition {
+    /// Returns if the [`ClusterCondition`] is considered to be in a good /
+    /// healthy state.
+    pub fn is_good(&self) -> bool {
```

Comment:
Just wanted to make sure it's explicitly made public. No need to comment ;)

Reply:
Yes. This allows outside code (for example `stackablectl`) to query this information. This is i.e. used for the colored output.